### ISSUE TITLE
FBAOutbound case issues

### ISSUE THREAD

**Issue by receta-erp** [jlevers/selling-partner-api#665]
## Problem description:
When trying to create an FBA Outbound request, we get errors asking for missing items in the data we are pushing across.  Looking at the model as shown in this git it's got the case naming convention different to the model as published by Amazon.  That seems to be common across the whole setup which generally would be fine (i.e the marketplace ID.  On this model it's stored as a value under marketplace_id, whereas on Amazon it's marketplaceId.  I'm guessing the code is doing that conversion on our behalf.

However the issue with this particular request is that it appears it's not doing the conversion.
So for example, when specifying a destination address, the postal code reference on this is shown as  "postal_code", whereas on the Amazon model it's PostalCode.

However, when I send across my request using postal_code it complains it cannot find the "PostalCode".  As soon as I change that on my side to match the Amazon naming convention - it seems to be ok with it.

So in short, the conversion between this code's naming conventions and the Amazon model is for the most part doing that auto conversion, but for some very specific metrics on this FBAOutbound - it appears to me that it's not.

## Error:
The following required parameters are missing: [Items.SellerFulfillmentOrderItemId, PostalCode, Items.SellerSKU]
```
# Your error here. PLEASE make sure to redact secrets from your error log!
```

## Code

**Comment by receta-erp** [comment]
Thanks for the speedy reply.  We will look into the updated version. As it stands we were able to figure out which particular metrics weren't playing ball and used the Amazon Model naming conventions for those specific metrics and it started to work without issue.  For reference those were:

in the deliveryaddress:
addressLine1
addressLine2
districtOrCounty
postalCode
countryCode

within items:
sellerSku
sellerFulfillmentOrderItemId

so it wasn't the "top level" array variables, it was the naming conventions within those inner arrays for the destination address and the items where the issue lay.

Hope that help?  I'll happily close this now as either our 'fix' will hopefully help or better still, your v6 covers it for most people.
